models/data_maps.go

```diff
@@ -298,3 +298,30 @@ func GetDataMapByGenesisHashAndChunkIdx(genesisHash string, chunkIdx int) ([]Dat
 
 	return dataMaps, err
 }
+
+func GetDataMap(genHash string, numChunks int) (dataMap []DataMap, vErr *validate.Errors) {
+
+	fileChunksCount := numChunks
+
+	if oyster_utils.BrokerMode != oyster_utils.TestModeNoTreasure {
```

Comment:
It looks like what you're doing here is building a datamap on the fly and not saving it to the DB?  That makes sense, though we might want to insert it (into a different table than data_maps) depending on how long the verification takes.  

Either way, you'll want to remove lines 306 through 308.  Those "modes" are for attaching new data_maps, not verifying old ones.

Reply:
Ok so we will ONLY save into db? Any idea about name of table?

So I will remove lines 306 through 308 + save data to db?

Reply:
Remove lines 306 through 308.  Let's discuss in slack whether we want the data in a table or just to do it on the fly.